README.md

```diff
@@ -21,7 +21,7 @@ The protobuf package of Google for proto3 is required for running this script.
 
 For printing QR codes, the qrcode module is required
 
-    pip install qrcode
+    pip install qrcode[pil]
```

Comment:
What is `[pil]` for?

Reply:
Installs the Pillow image library along with qrcode library to generate the image.
https://pypi.org/project/Pillow/